pjsip/include/pjsua2/endpoint.hpp

```diff
@@ -1870,6 +2063,15 @@ class Endpoint
 			  pj_status_t status,
 			  const pjsua_ip_change_op_info *info);
 
+    static pj_status_t pjsip_auth_create_aka_response_callback(
```

Comment:
minor: most callbacks in Endpoint has `on_` prefix, so perhaps better to rename  `pjsip_auth_create_aka_response_callback` --> `on_auth_create_aka_response()`?